### PR TITLE
added Grafeas Metadata tool for containers

### PR DESCRIPTION
**Pull request by vegasbrianc**
One more Tool which is hard to categorize as it is a new type of tool being developed by all the big cloud players.  As this a tool more for auditing of infrastructure it could fall into several categories.

Please assist with placing this in the correct section

Signed-off-by: Brian Christner <brian.christner@gmail.com>

https://github.com/grafeas/grafeas

This is a tool being developed by Google, Red Hat, IBM and more to address auditing and container supply chain management.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)
Yes
- The project submitted conforms to the quality standards outlined in the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)
Yes
- Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.

**RULES (because you probably didn't read the guidelines):**

Links will be tested with [awesome_bot](https://github.com/dkhamsing/awesome_bot). Please make sure your URL to be listed is unique (do some search queries), otherwise put a link at the end of `README.md` file.
 
- FORMAT
  - Use the following format: `[Name](Link) - Description.`
  - `[Name]` should be the title of the project
  - `(Link)` should be pointing to a GitHub repository, not to a website
  - `Description.` should end with a full stop/period
  - For GitHub repositories, add the name of the author with a link to its GitHub account after the description, like `By [@author](GitHub account)`
- ORDER: The entries are in lexical order (English letter order) 
- LABEL (Optional): Add one of the following icons next to the project name like ```[Name](Link) :icon: - Description.```
  - :heavy_dollar_sign: indicates if one needs a paying subscription to use the project.
  - :construction: indicates that the project is a work in progress/beta release.
  - :skull: indicates that the project is obsolete, but still worth mentioning it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [depcon](https://github.com/gondor/depcon) - Depcon is written in Go and allows you to easily deploy Docker containers to Apache Mesos/Marathon, Amazon ECS and Kubernetes.  By [@gonodr][gondor]
 * [deploy](https://github.com/ttiny/deploy) :skull: - Git and Docker deployment tool. A middle ground between simple Docker composition tools and full blown cluster orchestration by [@ttiny](https://github.com/ttiny)
 * [dockit](https://github.com/humblec/dockit) :skull: - Do docker actions and Deploy gluster containers! By [@humblec](https://github.com/humblec)
+* [Grafeas](https://github.com/Grafeas/Grafeas) - Grafeas defines metadata API spec for componentes. By [Grafeas](https://github.com/Grafeas)
```
Comment (veggiemonk) [inline README.md]: typo in `componentes` ... ?
Would it be possible to have a description that indicates a little bit what it does or what it is used for. It is totally unclear to me what is this project just by reading the description.

**Comment by vegasbrianc** [comment]
@veggiemonk It made sense at the time but re-reading it I also have no clue. 🔨  HAHA Thanks for catching it. Hope all is well and we should catch up soon!

File: README.md (hunk 1 above)
Comment (vegasbrianc) [reply README.md]: @veggiemonk This has been updated.

**Comment by veggiemonk** [comment]
@vegasbrianc sure thing... catchup anytime

**Pull request merged**